doc/segmentation_parameters.rst

```diff
@@ -8,35 +8,62 @@ A full list of parameters and descriptions can be found in the API Reference: :p
 
-:code:`level` can also be set to a `slice <https://docs.python.org/3/c-api/slice.html>`, which determines where in the vertical dimension (see `Vertical Coordinate`_) the features are seeded from. Note that :code:`level` operates in *array* coordinates rather than physical coordinates.
+:code:`level` can also be set to a `slice <https://docs.python.org/3/c-api/slice.html>`_, which determines where in the vertical dimension (see `Vertical Coordinate`_) the features are seeded from. Note that :code:`level` operates in *array* coordinates rather than physical coordinates.
+
+
+.. _seg_2d_feature_3d:
+
+===================================================
+Projecting 3D Spatial Features into 2D Segmentation
+===================================================
+When running feature detection on a 3D dataset and then using these detected features to segment data in 2D, the vertical coordinate is ignored. In case of vertically overlapping features, the larger :code:`Feature` value is currently seeded.
+
+
+.. _seg_3d_feature_3d:
+
+===================================================
+Projecting 3D Spatial Features into 3D Segmentation
+===================================================
+When running feature detection on a 3D dataset and then using these detected features to segment data in 3D, there are currently two options for determining how to seed the watershedding algorithm: *column* seeding (set by :code:`seed_3D_flag='column'`) and *box* seeding (set by :code:`seed_3D_flag='box'`). We generally recommend *box* seeding when running feature detection and segmentation in 3D.
+
+**Column** seeding (:code:`seed_3D_flag='column'`) works by setting seed markers throughout some or all of the vertical column at all detected feature centroids (i.e., one column per feature detected). While the default behavior is to seed throughout the full vertical column, the vertical extent of the seeds can be set by passing a `slice <https://docs.python.org/3/c-api/slice.html>`_ into the :code:`level` parameter. Note that :code:`level` operates in *array* coordinates rather than physical coordinates.
+
+**Box** seeding (:code:`seed_3D_flag='box'`) sets a cube or rectangular seed markers around the detected feature in 3D space. The marker size is user defined (in array coordinates) by :code:`seed_3D_size` as either an integer (for a cube) or a tuple of :code:`(int, int, int)`, ordered :code:`(vertical, hdim_1, hdim_2)`. Note that :code:`seed_3D_size` must be odd number to avoid the box becoming biased to one side. If two seed boxes overlap, the seeded area is marked with the closest feature centroid.
```

Comment:
missing 'an' from "Note that ... must be {an} odd number to avoid the box becoming biased to one side"

Reply:
Good catch, thanks!